src/fixvec/index.ts

```diff
@@ -1 +1,46 @@
-export default {}
+import { uint32Le, littleHexToInt, assertIsHexStr } from '../utils/index'
+
+export const serializeFixvec = (fixvec: HexString[]) => {
+  if (!Array.isArray(fixvec)) {
+    throw new TypeError(`Expect ${fixvec} to be an array`)
+  }
+
+  fixvec.forEach(item => {
+    assertIsHexStr(item)
+  })
+
+  const data = fixvec.map(item => (item.length % 2 ? `0${item.slice(2)}` : item.slice(2)))
+
+  const length = data[0]?.length ?? 0
```

Comment:
Length is an uint32 data

Reply:
The `length` here is used to make sure all items have the same length and is not included in the serialization.

Reply:
OK